Source/Core/BoxGeometry.js

```diff
@@ -111,8 +111,51 @@ define([
         var max = corner;
 
         var newOptions = {
-            minimumCorner : min,
-            maximumCorner : max,
+            minimum : min,
+            maximum : max,
+            vertexFormat : options.vertexFormat
+        };
+        return new BoxGeometry(newOptions);
+    };
+
+    /**
+     * Creates a cube from the dimensions of an AxisAlignedBoundingBox.
+     *
+     * @param {Object} options Object with the following properties:
+     * @param {AxisAlignedBoundingBox} options.boundingBox A description of the AxisAlignedBoundingBox.
+     * @returns {BoxGeometry}
+     *
+     * @exception {DeveloperError} AxisAlignedBoundingBox must be defined.
+     *
+     * @see BoxGeometry.createGeometry
+     *
+     * @example
+     * var aabb = Cesium.AxisAlignedBoundingBox.fromPoints(Cesium.Cartesian3.fromDegreesArray([
+     *      -72.0, 40.0,
+     *      -70.0, 35.0,
+     *      -75.0, 30.0,
+     *      -70.0, 30.0,
+     *      -68.0, 40.0
+     * ]));
+     * var box = Cesium.BoxGeometry.fromAxisAlignedBoundingBox({
+     *      boundingBox: aabb
+     * });
+     * var geometry = Cesium.BoxGeometry.createGeometry(box);
+     */
+    BoxGeometry.fromAxisAlignedBoundingBox = function(options) {
```

Comment:
This doesn't need to take `options`.  It can just take the bounding box object directly.